packages/next/client/performance-relayer.js

```diff
@@ -1,78 +1,54 @@
-function isTypeSupported(type) {
-  if (self.PerformanceObserver && PerformanceObserver.supportedEntryTypes) {
-    return PerformanceObserver.supportedEntryTypes.includes(type)
-  }
-  return false
-}
+import { getCLS, getFID, getFCP, getLCP, getTTFB } from 'web-vitals'
 
-export function observeLayoutShift(onPerfEntry) {
-  if (isTypeSupported('layout-shift')) {
-    let cumulativeScore = 0
-    const observer = new PerformanceObserver(list => {
-      for (const entry of list.getEntries()) {
-        // Only count layout shifts without recent user input.
-        if (!entry.hadRecentInput) {
-          cumulativeScore += entry.value
-        }
-      }
+export default onPerfEntry => {
+  // Measure Cumulative Layout Shift
+  getCLS(({ id, value }) =>
+    onPerfEntry({
+      name: 'cumulative-layout-shift',
+      value: value * 1000,
```

Comment:
I multiply by 1000 in the [example for Google Analytics](https://github.com/GoogleChrome/web-vitals#send-the-results-to-google-analytics), specifically because GA requires all metrics to be integers (for some reason) and I need more precision. If this is being used for non-GA cases, I wonder if it makes sense to keep the same value as exposed by the API.

Reply:
Good point, thought maybe it'll be better to match precision closer to our documentation but agreed it's probably better to use the same values returned by the API